api/specs/web-server/_license_goods.py

```diff
@@ -0,0 +1,60 @@
+""" Helper script to generate OAS automatically
+"""
+
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+
+from typing import Annotated
+
+from _common import as_query
+from fastapi import APIRouter, Depends, status
+from models_library.api_schemas_webserver.license_goods import LicenseGoodGet
+from models_library.generics import Envelope
+from models_library.rest_error import EnvelopedError
+from simcore_service_webserver._meta import API_VTAG
+from simcore_service_webserver.licenses._exceptions_handlers import _TO_HTTP_ERROR_MAP
+from simcore_service_webserver.licenses._models import (
+    LicenseGoodsBodyParams,
+    LicenseGoodsListQueryParams,
+    LicenseGoodsPathParams,
+)
+
+router = APIRouter(
+    prefix=f"/{API_VTAG}",
+    tags=[
+        "licenses",
+    ],
+    responses={
+        i.status_code: {"model": EnvelopedError} for i in _TO_HTTP_ERROR_MAP.values()
+    },
+)
+
+
+@router.get(
+    "/license-goods",
```

Comment:
Why "license" and not "licensed" ?

Reply:
Yes, you are right! 👍 I will wait some time until everyone shares their opinion and then change it once.

Reply:
UPDATE: I like Pedros's proposal to change it to: `/catalog/licensed-items` I will do that now.